src/App.css

```diff
@@ -90,30 +92,32 @@ a:active { text-decoration: none; }
 .icon {
   width: 2.813rem;
   height: 2.813rem;
-  margin: 0 0 0 41.5rem;
+  margin-left: auto;
   padding: 0.875rem;
   border: solid 1px var(--white);
 }
 
 .BookC {
```

Comment:
- [ ] As suggested by the previous reviewers, the borders are not the same color as compared to the the Given design on Zeplin:

| Your Design  |  Zeplin Design |
| ------------- | ------------- |
| <img width="179" alt="image" src="https://github.com/OnlyUnknown/book-store/assets/52934128/89d31a72-6d27-4789-a455-756eed347c8f">  | <img width="200" alt="image" src="https://github.com/OnlyUnknown/book-store/assets/52934128/4a88ce9f-c9ed-4154-84d4-01f7e5f6c89d"> |
| Content Cell  | Content Cell  |